src/model/Weather.js

```diff
@@ -193,6 +196,7 @@ export default class Weather{
     /**
      * Static constant: cloudiness levels
      */
+    // CODE_REVIEW Podobają mi się te improwizowane enumy.
```

Comment:
Mnie się bardziej podoba, że typescript ma prawdziwe :D Tak bardzo mi ich brakowało, jak to pisałem ;)